ttnn/cpp/ttnn/operations/reduction/prod/prod.cpp

```diff
@@ -15,7 +15,6 @@
 
 namespace ttnn::operations::reduction {
 
-namespace prod {
```

Comment:
where is the register_operation call for this?

Reply:
in prod.hpp